Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift

```diff
@@ -3,21 +3,6 @@ import Library
 import Prelude
 import UIKit
 
-private enum Layout {
-  enum Card {
-    static let width: CGFloat = 240
-  }
-
-  enum ImageView {
-    static let width: CGFloat = 64
-    static let height: CGFloat = 40
-  }
-
-  enum Button {
-    static let width: CGFloat = 217
-  }
-}
-
 final class PledgeCreditCardView: UIView {
```

Comment:
Since the name of properties now have the prefix `paymentSource`, should we rename this file to `PledgePaymentSourceView`?

Reply:
I renamed these to `cardStyles` so it's more consistent with the naming of our files.